pom.xml

```diff
@@ -135,6 +136,17 @@
         <artifactId>byte-buddy</artifactId>
         <version>1.12.10</version>
       </dependency>
+      <!-- Not everything needs revApi or the reporter -->
+      <dependency>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-java</artifactId>
+        <version>${revapi-java.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-reporter-json</artifactId>
+        <version>${revapi-reporter-json-version}</version>
+      </dependency>
```

Comment:
Why is that needed? The versions given in `build` will override these values.

Reply:
I forgot to talk with you about this topic. Is it possible to declare the java and reporter dependency in this pom so that we don't need to declare it again in every child pom? Would be even better for that analysis pom which get used quite often as parent.